lib/content_block_tools/presenters/base_presenter.rb

```diff
@@ -56,7 +57,13 @@ def default_content
       end
 
       def content_for_fields
-        content_block.details.deep_symbolize_keys.dig(*field_names)
+        content = content_block.details.deep_symbolize_keys.dig(*field_names)
+        if content.blank?
+          ContentBlockTools.logger.warn("Content not found for content block #{content_block.content_id} and fields #{field_names}")
```

Comment:
Is it worth adding tests for the logger too?

Reply:
I debated it but wasn't sure how much value it would add? But do you think it's worth unit testing e.g. that the logger was called with what's expected? Or test the initialisation bit?

Reply:
Yeah, it was more testing that the log messages were being sent, more to document the functionality than anything else

Reply:
cool, have added now